### PR TITLE
Add info about posting questions and problems

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -178,6 +178,14 @@ Contributing to DALI
 
 Contributions to DALI are more than welcome. To make the pull request process smooth, please follow these `guidelines <CONTRIBUTING.md>`_.
 
+Reporting problem, asking questions
```
Comment (szalpal) [inline README.rst]: Typo: I suppose, there should be "problems"

File: README.rst [hunk 2]
```diff
@@ -178,6 +178,14 @@ Contributing to DALI
...
+-----------------------------------
+
+We appreciate any feedback, question or bug reported regarding this project. To make the process of answering to all those requests smoother please stick to following the guide (it is based on https://stackoverflow.com/help/mcve) when help with code is needed. So posted examples should be:
```
Comment (szalpal) [inline README.rst]: "to following the guide" -> "to the following guide"

Comment (klecki) [inline README.rst]: "We appreciate any feedback, question or bug" -> "questions or bugs".

File: README.rst (hunk 1 above)
Comment (JanuszL) [reply README.rst]: Done

File: README.rst (hunk 2 above)
Comment (JanuszL) [reply README.rst]: Done

Comment (JanuszL) [reply README.rst]: Done

**Comment by JanuszL** [comment]
@cliffwoolley was probably sucked into something more important. I will ask Lynsey for help with the wording.

**Comment by JanuszL** [comment]
@cliffwoolley - I have applied all Lynsey suggestions. So are we good to go with that?

**Pull request merged**